compose/src/main/kotlin/com/yourssu/handy/compose/SnackBar.kt

```diff
@@ -0,0 +1,215 @@
+package com.yourssu.handy.compose
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.core.tween
+import androidx.compose.animation.expandVertically
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.animation.shrinkVertically
+import androidx.compose.animation.slideOutVertically
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Popup
+import com.yourssu.handy.compose.foundation.ColorGray800
+import com.yourssu.handy.compose.foundation.ColorStatusRedSub
+import com.yourssu.handy.compose.foundation.HandyTypography
+import kotlinx.coroutines.delay
+
+/**
+ * 정보성 스낵바의 UI를 그린 함수입니다.
+ *
+ * @param text 스낵바의 문구를 나타내는 텍스트, 최대 두 줄까지 입력 가능
+ * @param modifier Modifier
+ */
+@Composable
+fun InfoSnackBarItem(
```

Comment:
InfoSnackBarItem, InfoSnackBar 라는 명명보다는 
UI 컴포넌트는 InfoSnackBar, 사용해야 하는 구현체는 InfoSnackBarHost라고 명명하는건 어떨까요?

조만간 HandyScaffold를 구현할 예정인데 그때 
`To show a [Snackbar], use [SnackbarHostState.showSnackbar].
`SnackbarHostState내 show, dismiss등등을 사용하는 방식이 좋을 것 같아서요
[참고](https://developer.android.com/develop/ui/compose/components/snackbar#snackbar_with_action)


그리고 scaffold 도입하여 snackbar를 사용하게 된다면 현재 popUp 방식보다는 
coroutine mutex를 활용하는 건 어떨까요?
[YDS 참고](https://github.com/yourssu/YDS-Android/blob/develop/compose/src/main/java/com/yourssu/design/system/compose/component/toast/ToastHost.kt)
[material 참고](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/SnackbarHost.kt)

Mutex 사용이유는 
Mutex in order to prevent value loss due to concurrency
라고 하네용
[참고](https://gist.github.com/gab-stargazer/e66b2bf4d954847ce76164215f9dc531#the-solution)